Externals_CLM.cfg

```diff
@@ -1,8 +1,8 @@
 [fates]
 local_path = src/fates
 protocol = git
-repo_url = https://github.com/NGEET/fates
-tag = sci.1.21.0_api.7.0.0
+repo_url = https://github.com/NCAR/fates-release
+tag = fates_s1.23.0_a7.1.0
```

Comment:
@ekluzek , our current "most-stable" version is sci.1.21.0_api.7.0.0

We are awaiting some fixes to come through and update fates_s1.23.0_a7.1.0 to 
fates_s1.24.1_a7.1.0

Although, both tags mentioned here should pass the same number of tests.

Reply:
i'm also noticing here that you are pointing to the NCAR repo.  SHouldn't we keep fates_next_api pointing towards NGEET/fates ?

Reply:
Normally I think fates_next_api should point to NGEET/fates. I'm pointing to the NCAR repo right now, so I can show that it works on hobart. There are some changes on the NCAR repo, for that purpose that I want to propose to move to NGEET/fates. So we can either get these into NGEET/fates and point to that version, or we can point to the NCAR repo, just for this next version. Which do you prefer?

Reply:
The sci.1.21.0_api.7.0.0 tag didn't run the fates test list because of a problem with the fates parameter file. So I had to move the fates version forward. Although I could also try a different version of fates if you want me to. Should I try a different one?

Reply:
hmm, I guess I don't feel too strongly about where the tag is hosted. If that is the case, then I guess just use whichever makes your workflow straightforward.

Reply:
> The sci.1.21.0_api.7.0.0 tag didn't run the fates test list because of a problem with the fates parameter file. So I had to move the fates version forward. Although I could also try a different version of fates if you want me to. Should I try a different one?

ok, I see.  The fates parameter default file has been updated since that tag.

The newest tag while not the most stable, fails during long simulations), should match the fates parameter file on the HEAD of fates_next_api, and should pass the tests.